docker-compose.yml

```
@@ -93,4 +95,60 @@ services:
       - "8000:8000"
     depends_on:
       - embedding
-      - llm 
\ No newline at end of file
+      - llm
+
+  vocab-importer:
+    build: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)